src/installLogsPrinter.js

```diff
@@ -73,8 +74,13 @@ function installLogsPrinter(on, options = {}) {
           options.printLogsToFile === "always" ||
           isHookAndShouldLog
         ) {
+          const outputFileMessages =
+            typeof options.outputCompactLogs === 'number' && options.outputCompactLogs >= 0
```

Comment:
In order to not introduce a breaking change let's default to main `compactLogs` option here if this is undefined

Reply:
feel free to refactor if you'd rather have it written differently

Reply:
I feel like the main feature you require is not possible yet: How would one disable compacting the logs for file output if compacting is enabled with the main option? Cause if you set it to 0 or more it will compact it, and if you set it to something else it will use the previously compacted logs.

Maybe I don't see something here, correct me if I wrong. Maybe we also need a testcase with this case.

Reply:
Gotcha. Made some updates:
```
let outputFileMessages
if (typeof options.outputCompactLogs === 'number') {
    if (options.outputCompactLogs >= 0) {
        outputFileMessages = compactLogs(messages, options.outputCompactLogs)
    } else {
        outputFileMessages = messages // if -1, override compactLogs, don't compact
    }
} else {
    outputFileMessages = terminalMessages // if unspecified, go with compactLogs
}
```

I realized that we need a way to use `outputCompactLogs` override when the user wants to compact the terminal logs but not the output file logs, so I made the minimum on `outputCompactLogs` -1 . Setting to -1 causes all logs to go to output file even if `compactLogs` specified for the terminal.

Added tests for this to `outputCompactLogs.spec.js`